src/Microsoft.AspNetCore.Http.Connections.Client/Internal/LongPollingTransport.Log.cs

```diff
@@ -99,6 +108,30 @@ public static void PollResponseReceived(ILogger logger, HttpResponseMessage resp
                         response.Content.Headers.ContentLength ?? -1, null);
                 }
             }
+
+            public static void SendingDeleteRequest(ILogger logger, Uri pollUrl)
+            {
+                if (logger.IsEnabled(LogLevel.Debug))
+                {
+                    _sendingDeleteRequest(logger, pollUrl.ToString(), null);
```

Comment:
Does the logging not ToString Uri's automatically? I'm sure I have seen logging that passes the Uri directly.

Reply:
Yep, we should be doing that.